src/components/Nav/Nav.js

```diff
@@ -74,6 +78,55 @@ function Nav() {
     return () => document.removeEventListener('mousedown', handleClickOutside)
   })
 
+  useEffect(() => {
+    if (router.pathname === '/') {
```

Comment:
Long if else block can be refactored to use an object for conditions and dry up this useEffect.
```suggestion
    const pathNameMap = {
      '/': '/#our_stories',
      '/about': '/about#about',
      '/board': '/board#about',
      ...
    }
    
    if (pathNameMap[router?.pathname]) {
      setMainContentLink(pathNameMap[router.pathname])
    }
```